front/src/pages/PrivatePages/sharedPrivateComponents/menu/MenuResponsive.jsx

```diff
@@ -1,25 +1,25 @@
-import React, { useContext } from 'react';
-import { Icon } from '@iconify/react';
-import ProfilePictureHeader from '../../../../assets/admins/ProfilePictureHeader.svg';
-import MenuComponent from './MenuComponent';
-import Logout from '../../../../assets/icons/privatePage/Icon_logout.svg';
-import { Link, useNavigate } from 'react-router-dom';
-import axios from 'axios';
-import Context from '../../../../context/Context';
+import React, { useContext } from "react";
+import { Icon } from "@iconify/react";
+import ProfilePictureHeader from "../../../../assets/admins/ProfilePictureHeader.svg";
+import MenuComponent from "./MenuComponent";
+import Logout from "../../../../assets/icons/privatePage/Icon_logout.svg";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import Context from "../../../../context/Context";
 
 export default function MenuResponsive({ menu, setMenu }) {
   const navigate = useNavigate();
   const { deslogearme } = useContext(Context);
```

Comment:
Tendría que estar así ahora 
```suggestion
  const { logOut} = useContext(Context);
```